### PR TITLE
Cleanup and add systemd unit file

### PR DESCRIPTION
**Pull request by nhooyr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ Previously maintained by [KSXGitHub](https://github.com/KSXGitHub)
 ## Updating
 
 If there are no breaking changes with the release, updating the package is as
-simple as updating `pkgver` in `PKGBUILD` to the new version.
+simple as updating `pkgver` in `PKGBUILD` to the new version. Make sure
+to run `updpkgsums` to update the sha256 sums.
```
Comment (coadler) [inline README.md:12]: This helps a lot.

File: PKGBUILD [hunk 2]
```diff
@@ -2,23 +2,49 @@
...
+)
 
 package() {
-  # Special thanks to @ben0mega and @ava1ar
-  cd "$srcdir"/"${_dirname}"
-  find . -type f -exec install -Dm 755 "{}" "$pkgdir/opt/$pkgname/{}" \;
-  mkdir -p $pkgdir/usr/bin/
-  ln -s /opt/$pkgname/bin/code-server $pkgdir/usr/bin/$pkgname
-  install -Dm644 LICENSE.txt "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
+  if [[ $(uname -m) == x86_64 ]]; then
```
Comment (coadler) [inline PKGBUILD:33]: It seems like pacman injects `$CARCH` for checking the target arch. Is there a reason we're not using that?

File: PKGBUILD [hunk 3]
```diff
@@ -2,23 +2,49 @@
...
+    release_name+=-amd64
+  else
+    release_name+=-amd64
```
Comment (coadler) [inline PKGBUILD:36]: These are the same. Did you mean `arm64`?

File: PKGBUILD (hunk 2 above)
Comment (nhooyr) [reply PKGBUILD:33]: `uname -m` is well known and we use it everywhere else so I figured it's best to keep the code clear.

File: PKGBUILD (hunk 3 above)
Comment (nhooyr) [reply PKGBUILD:36]: Mhmm good catch.

**Pull request merged**